console/billing/quickstart.mdx

```diff
@@ -0,0 +1,76 @@
+---
+meta:
+  title: Billing - Quickstart
+  description: This page shows you how to get started with the billing of your Scaleway account.
+content:
+  h1: Billing - Quickstart
+  paragraph: This page shows you how to get started with the billing of your Scaleway account.
+tags: billing payment 
+dates:
+  validation: 2024-02-20
+  posted: 2024-02-20
+categories:
+  - billing
+---
+
+## How to add a payment method
+
+Before you can order Scaleway resources, you must add your payment method to your account.
+
+<Macro id="iam-requirements" />
+
+1.Click **Billing** in the Organization drop-down menu to go to the [Billing page](https://console.scaleway.com/organization/billing).
+2. Click **Payment and billing**.
+3. Scroll down to **Payment Methods**.
+
+You can choose between two payment methods: **credit card** or **SEPA mandate**.
+
+### How to add a credit card
+
+1. Enter the details of your credit or debit card.
+2. Click **Add credit card** to confirm. A [3D Secure](https://en.wikipedia.org/wiki/3-D_Secure) validation is initiated. Depending on your bank, you may be asked to enter a code received by SMS or via an application on your smartphone.
+3. Enter the code and validate the form.
+    <Message type="note">
+     When you register a new credit card, an authorization charge of **1€** is debited from your account. This **Authorization Charge** is a dummy transaction made to check the validity of the card and will carry a 4-digit validation code, displayed on your bank account statement. The authorization charge will be reimbursed within 48 to 72 hours.
+    </Message>
+4. Click **Verify**. A pop-up appears.
+5. Enter the 4-digit validation code displayed on your bank account statement.
+    <Message type="note">
+      Bank account statements have different displays depending on the bank, but the validation code can always be found in the description of your transactions. The image below provides an example of how the validation code might appear in your bank statement:
+
+      <Lightbox src="scaleway-bank-statement.webp" alt="Screenshot of a bank account statement. Under the transactions section, an entry called SCW*C-6569E-COMMERCE displays. The validation code is represented by the number sequence" />
+    </Message>
+
+
+<Message type="important">
+  You have up to 10 days to enter your validation code in the console.
```

Comment:
Maybe explain what happens if you dont.

Reply:
> Maybe explain what happens if you dont.

I don't know what happens if they don't enter the validation code :) Could you please send it to me?

Reply:
Nothing happens for the users. It only triggers an internal alert for us (Trust & Safety team) but won't prevent users from creating resources. However, we want to encourage them to enter this code for security matters. @pshambulingappa I can show you the admin console if you want to understand better how we "rate" our clients regarding security :) 

We can write: 
```suggestion
  You have up to 10 days to enter your validation code in the console. This helps us secure your account. 
```